### PR TITLE
pkgdown upgrades with diagrams and more documentation

### PR DESCRIPTION
**Pull request by fontikar**
Addresses #65 
- Vignette is prerendered for speedier testing local and on GHA
- Customised pkgdown website with bootswatch theme
- Customisation with articles menu
- Inserted figures made with powerpoint in vignette and data provider article
- Updated README to point to pkgdown

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -23,11 +23,14 @@ library(ausflora)
...
+# ausflora <img src="man/figures/ausflora_hex2.png" align="right" width="120"/>
 
-A  R package for aligning and updating taxonomic names from the [Australian Plant Census (APC)](https://biodiversity.org.au/nsl/services/search/taxonomy)
+'ausflora' uses the [Australian Plant Census (APC)](https://biodiversity.org.au/nsl/services/search/taxonomy) and [Australian Plant Name Index](https://biodiversity.org.au/nsl/services/search/names) to standardise Australian plant taxon names. 'ausflora' also supplies information about
+the established status of plant taxa across different states/territories.
 
-## Installation 
+## Installation
+
+'ausflora' is current not on CRAN. Install the current development version:
```
Comment (dfalster) [inline README.Rmd:33]: currently

File: vignettes/ausflora.Rmd [hunk 2]
```diff
@@ -1,97 +1,134 @@
...
+
+```r
 resources <- load_taxonomic_resources(stable_or_current_data = "stable", version = "0.0.2.9000")
+#> Loading resources......done
 ```
 
 ## Standardise plant taxon names
 
-Now we can query our taxonomic names against the taxonomic resources we just retrieved using `create_taxonomic_update_lookup()`. This all-in-one function will:
+Now we can query our taxonomic names against the taxonomic resources we just retrieved using `create_taxonomic_update_lookup()`. This is an all-in-one function will:
```
Comment (dfalster) [inline vignettes/ausflora.Rmd:101]: if modifying as prpoposed, you need to also change "will" -> "that will"

File: vignettes/ausflora.Rmd [hunk 3]
```diff
@@ -100,30 +137,61 @@ The `apc_name` is the currently, accepted taxon name used by the Australian Plan
...
+#> # ℹ 26,237 more rows
 ```
 
 Here is a breakdown of all possible values for `origin` 
 
-```{r}
+
+```r
 library(purrr)
 library(janitor)
```
Comment (dfalster) [inline vignettes/ausflora.Rmd:196]: So we require janitor?

**Review by dfalster — APPROVED** [review]
Very nice! 

Happy for you to merge once suggested minor changes are implemented 

No need for additional review

File: vignettes/ausflora.Rmd (hunk 2 above)
Comment (fontikar) [reply vignettes/ausflora.Rmd:101]: Good spot!

File: vignettes/ausflora.Rmd (hunk 3 above)
Comment (fontikar) [reply vignettes/ausflora.Rmd:196]: I've popped into Suggests it is for [development tasks](https://r-pkgs.org/description.html#sec-description-imports-suggests) so it won't get installed unless user specifies to do so!

**Review by dfalster — APPROVED** [review]

**Pull request merged**